script/packtpub.py

```diff
@@ -124,6 +124,10 @@ def download_ebooks(self, types):
             for type in types]
 
         directory = self.__config.get('path', 'path.ebooks')
+
+        if self.__config.get('path', 'path.separate') == "true":
+            directory = directory + '/' + self.info['title'] + ' - ' + self.info['author']
```

Comment:
* use regexp on `title` and `author` to remove any extraneous character: restrict to [a-zA-Z] for example
* try to avoid any space, I suggest to use `_` as separator
* maybe verify that if the folder is nested for some reason, the script doesn't throw any error (equivalent of unix `mkdir -p /my/path/to/new/folder`
* I'm not a python ninja, so don't know if `XXX == "true"` is the best way for comparison or you prefer to check simply if `path.separate.extras` is not empty